src/components/InviteUsers.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <v-card class="no-users">
+    <v-card-title>
+      <v-row>
+        <v-col class="py-0">
+          <h3>No users yet</h3>
+        </v-col>
+        <v-col align="end" class="py-0">
+          <v-btn icon @click="$root.$emit('closeInvitations')">
```

Comment:
I don't really like using `$root.$emit`, it implies magic and is not easily debugged

if possible please change this or change my mind :wink: